codespell_lib/data/dictionary.txt

```diff
@@ -6116,6 +6120,7 @@ doesnt->doesn't, does not,
 doesnt;->doesn't
 doess->does
 doestn't->doesn't
+dof->of
```

Comment:
Also suggest doff as in "doff your hat" ( https://www.phrases.org.uk/meanings/doff-your-hat.html )

Reply:
Done